OpenRA.Game/Server/Server.cs

```diff
@@ -1166,12 +1166,12 @@ public void DropClient(Connection toDrop)
 				if (State == ServerState.GameStarted)
 				{
 					if (dropClient.IsObserver)
-						SendLocalizedMessage(ObserverDisconnected, Translation.Arguments("player", dropClient.Name));
+						SendLocalizedMessage(ObserverDisconnected, ("player", dropClient.Name));
 					else
-						SendLocalizedMessage(PlayerDisconnected, Translation.Arguments("player", dropClient.Name, "team", dropClient.Team));
+						SendLocalizedMessage(PlayerDisconnected, ("player", dropClient.Name), ("team", dropClient.Team));
```

Comment:
This syntax IMO doesn't feel right by OpenRA's conventions. I think it would probably better to pass these as a dictionary (like we do for WidgetArgs) and convert to the format Linguini expects deeper inside our plumbing.
 
```csharp
SendLocalizedMessage(PlayerDisconnected, new TranslationArguments()
{
    {"player", dropClient.Name},
    {"team", dropClient.Team}
});
```

Reply:
I went for this because it shortens the lines with translation strings and arguments.

Reply:
Reverted back to the previous dictionary approach. Internally, Linguini uses an arguments dictionary anyway.

Reply:
> I think it would probably better to pass these as a dictionary (like we do for WidgetArgs) and convert to the format Linguini expects deeper inside our plumbing.

I don't understand why you changed this in #21602 then.